perfume-api/src/main/java/io/perfume/api/sample/domain/Sample.java

```diff
@@ -0,0 +1,71 @@
+package io.perfume.api.sample.domain;
+
+import jakarta.persistence.*;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.AvoidStarImportCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Using the '.*' form of import should be avoided - jakarta.persistence.*.